tests/core/test_solve.py

```diff
@@ -1576,6 +1572,23 @@ def test_timestamps_1():
         )
         assert convert_to_dist_str(link_dists) == order
 
+def test_channel_priority_churn_minimized():
```

Comment:
This is a regression test for churn observed with 4.7.1.  If you started with just defaults, did an install with ``-c conda-forge``, you'd update to a lot of packages.  On the next operation, it would go back to defaults.